eng/common/scripts/Update-DocsMsToc.ps1

```diff
@@ -116,7 +105,8 @@ $onboardedPackages = &$GetOnboardedDocsMsPackagesFn `
 $metadata = (Get-CSVMetadata).Where({
     $_.Package `
       -and $onboardedPackages.ContainsKey($_.Package) `
-      -and $_.Hide -ne 'true'
+      -and $_.Hide -ne 'true' `
+      -and $_.New -eq 'true'
```

Comment:
We shouldn't filter by `$_.New -eq 'true'` here. 

We're not generating the ToC based entirely on what's in metadata. We start with the list of packages that are onboarded and then we add metadata information to those onboarded packages... Whether or not a package is `New` doesn't affect if a package is already onboarded in the docs (usually under the `Legacy` moniker)